### DISCUSSION TITLE
Does rspamd transfer the emails to external service?

### DISCUSSION THREAD

**Discussion by alexletterws** [rspamd/rspamd#4382] [General]
Hello everyone, I am new to rspamd.
When an email reaches my server, postfix passes it to rspamd, then rspamd evaluates the spam score for it.
My question is, does rspamd send the email sender address and body to scan it at some external service?
Or, is the email never transferred outside of my server?
Thanks in advance!

**Comment by fatalbanana** [comment]
Rspamd does not copy mail somewhere (it can be configured to do something like this using `metadata_exporter` for e.g.).

In the default configuration it queries various RBLs for IP addresses/domain names/hashes/etc and Rspamd.com for fuzzy hashes.

https://rspamd.com/doc/quickstart.html#configuring-rbls
https://www.rspamd.com/doc/modules/fuzzy_check.html
https://rspamd.com/doc/usage_policy.html
https://www.rspamd.com/doc/modules/rbl.html
